pkg/agent/monitortool/monitor.go

```diff
@@ -15,21 +15,26 @@
 package monitortool
 
 import (
+	"context"
 	"math/rand"
 	"net"
 	"sync"
 	"sync/atomic"
 	"time"
 
+	stv1aplpha1 "antrea.io/antrea/pkg/apis/stats/v1alpha1"
+	"antrea.io/antrea/pkg/util/env"
```

Comment:
These imports should be moved to the third import block

Reply:
Thanks, it is fixed now.